presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -460,6 +467,40 @@ public void testLargeFile(ReadMode readMode)
                 });
     }
 
+    @Test
+    public void testFileSystemBindings()
+            throws Exception
+    {
+        initializeRubix(new RubixConfig());
+        try (FileSystem fileSystem = getCachingFileSystem(context, new Path("s3://bucket_name"))) {
```

Comment:
can you also add entries for `s3n`, `s3a`, `wasb`, `wasbs` and `hdfs`?

Reply:
> can you also add entries for s3n, s3a, wasb, wasbs and hdfs?

Done. Added comment for `wasbs` as it requires valid credentials